website/docs/repository.html.markdown

```diff
@@ -6,21 +6,21 @@ description: |-
 
 ---
 
-# Resource: helm_repository
+# Data Source: helm_repository
```

Comment:
In other providers documentation, datasources are usually placed in a `d` folder, separate from resources which are in an `r` folder.

This is mostly important when both a datasource and a resource exist with the same name, e.g. https://www.terraform.io/docs/providers/kubernetes/d/service.html vs https://www.terraform.io/docs/providers/kubernetes/r/service.html